jsk_panda_robot/jsk_panda_startup/launch/panda_control.launch

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" ?>
+
+<!-- this file is modified version of franka_control.launch -->
+<!-- https://github.com/frankaemika/franka_ros/blob/30e598aa6fb703cc80a203481e6427f397337b4c/franka_control/launch/franka_control.launch -->
+
+<launch>
+  <arg name="robot_ip" default="192.168.0.2"/>  <!-- add ip -->
+  <arg name="robot" default="panda" doc="choose your robot. Possible values: [panda, fr3]"/>
+  <arg name="arm_id" default="$(arg robot)" />
+  <arg name="load_gripper" default="true" />
+  <arg name="xacro_args" default="" />
+
+  <param name="robot_description" command="$(find xacro)/xacro $(find panda_eus)/models/panda.urdf.xacro hand:=$(arg load_gripper) arm_id:=$(arg arm_id) $(arg xacro_args)"/>  <!-- change xacro -->
+
+  <include file="$(find franka_gripper)/launch/franka_gripper.launch" if="$(arg load_gripper)">
+    <arg name="robot_ip" value="$(arg robot_ip)" />
+    <arg name="arm_id"   value="$(arg arm_id)" />
+  </include>
+
+  <node name="franka_control" pkg="franka_control" type="franka_control_node" output="screen" required="true">
+    <rosparam command="load" file="$(find franka_control)/config/franka_control_node.yaml" subst_value="true" />
+    <param name="robot_ip" value="$(arg robot_ip)" />
+  </node>
+  <rosparam command="load" file="$(find franka_control)/config/default_controllers.yaml" subst_value="true" />
+  <node name="state_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="franka_state_controller position_joint_trajectory_controller" />  <!-- add controller -->
```

Comment:
In addition to add `position_joint_trajectory_controller` , change the path of xacro to load from `panda_eus` dir,  and add default IP. (comments for modified parts are left in the launch file)

I thought reuse of the code as you mentioned, but I'm not sure it will be correctly overridden. So, I copied the `franka_control.launch` which is not so much volume and left comments.
In order to separate core and parameters, we might need to make something like `franka_robot.launch` based on `franka_control.launch`, and then `panda.launch` can be described with only parameters  of `franka_robot.launch`, such as ip, xacro, controllers etc..

It is ok to change the name as following your preferences. `panda_control.launch` just came from `franka_control.launch`.
[dfa63af](https://github.com/jsk-ros-pkg/jsk_robot/pull/1799/commits/dfa63af836818f3ea5bd52a5cee6b9459e6b0506)

Reply:
> change the path of xacro to load from panda_eus dir, and add default IP. (comments for modified parts are left in the launch file)

Could you try the following?
```
<launch>
  <arg name="robot_ip" default="192.168.0.2"/>
  <!-- Specify other arguments here -->

  <include file="$(find franka_control)/launch/franka_control.launch>
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <!-- Specify other arguments here -->
  </include>
  <param name="robot_description" command="$(find xacro)/xacro $(find panda_eus)/models/panda.urdf.xacro hand:=$(arg load_gripper) arm_id:=$(arg arm_id) $(arg xacro_args)"/>

  <node name="position_joint_trajectory_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="position_joint_trajectory_controller" />

</launch>
```
Maybe you can overwrite `robot_description` by setting it after including `franka_control.launch`.
See https://answers.ros.org/question/199608/roslaunch-order-of-rosparams/ for details.

If this does not work, `panda.launch` without including `franka_control.launch` is OK, but at least, `state_controller_spawner` is a wrong name because it spawns not only `franka_state_controller` but also `position_joint_trajectory_controller`.
I recommend to add `position_joint_trajectory_controller_spawner` separately.

Reply:
I tried and the overwrite of parameters seems working fine.
https://github.com/jsk-ros-pkg/jsk_robot/pull/1799/commits/3a79e150c8f174dcc34241ba5c556359c52a27be

Thank you for the link and tips of rosparam!